lib/pcb/layer-name-to-color.ts

```diff
@@ -12,3 +12,15 @@ export function layerNameToColor(layerName: string) {
     "white"
   )
 }
+export const SOLDER_PASTE_LAYER_NAME_TO_COLOR = {
+  bottom: "rgb(169, 169, 169)", // Lighter gray (Light gray)
+  top: "rgb(105, 105, 105)", // Darker gray (Dim gray)
+}
+
+export function solderPasteLayerNameToColor(layerName: string) {
+  return (
+    SOLDER_PASTE_LAYER_NAME_TO_COLOR[
+      layerName as keyof typeof SOLDER_PASTE_LAYER_NAME_TO_COLOR
+    ] ?? "gray" // Gray color for solder paste
```

Comment:
I think we want to make it always black, on any layer, since you export the layers separately and theyre fed into a laser cutter machine